tests/bluetooth/tester/overlay-mesh-v1d1.conf

```diff
@@ -0,0 +1,30 @@
+CONFIG_ENTROPY_GENERATOR=y
```

Comment:
Shouldn't good enough entropy be set for mesh 1.0.1 tests either?

Reply:
Hm, but mesh1.1 is compiled for `native_posix` too.

Reply:
Remove this and try to compile

Reply:
Added entropy generator to mesh overlay:
```
diff --git a/tests/bluetooth/tester/overlay-mesh.conf b/tests/bluetooth/tester/overlay-mesh.conf
index 840af06c1c..8ac2b8c59d 100644
--- a/tests/bluetooth/tester/overlay-mesh.conf
+++ b/tests/bluetooth/tester/overlay-mesh.conf
@@ -1,3 +1,5 @@
+CONFIG_ENTROPY_GENERATOR=y
+
 CONFIG_BT_MESH=y
 CONFIG_BT_MESH_RELAY=y
 CONFIG_BT_MESH_PB_ADV=y

```
result:
```
alkh-local@alkh-linux:~/ncs/zephyr/tests/bluetooth/tester$ west build -b native_posix -- -DOVERLAY_CONFIG="overlay-mesh.conf"
-- west build: generating a build system
Loading Zephyr default modules (Zephyr base).
-- Application: /home/alkh-local/ncs/zephyr/tests/bluetooth/tester
-- CMake version: 3.27.4
-- Found Python3: /usr/bin/python3 (found suitable version "3.10.12", minimum required is "3.8") found components: Interpreter 
-- Cache files will be written to: /home/alkh-local/.cache/zephyr
-- Zephyr version: 3.4.99 (/home/alkh-local/ncs/zephyr)
-- Found west (found suitable version "1.0.0", minimum required is "0.14.0")
-- Board: native_posix
-- Found host-tools: zephyr 0.16.1 (/home/alkh-local/zephyr-sdk-0.16.1)
-- Found toolchain: host (gcc/ld)
-- Found Dtc: /home/alkh-local/zephyr-sdk-0.16.1/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.6.0", minimum required is "1.4.6") 
-- Found BOARD.dts: /home/alkh-local/ncs/zephyr/boards/posix/native_posix/native_posix.dts
-- Generated zephyr.dts: /home/alkh-local/ncs/zephyr/tests/bluetooth/tester/build/zephyr/zephyr.dts
-- Generated devicetree_generated.h: /home/alkh-local/ncs/zephyr/tests/bluetooth/tester/build/zephyr/include/generated/devicetree_generated.h
-- Including generated dts.cmake file: /home/alkh-local/ncs/zephyr/tests/bluetooth/tester/build/zephyr/dts.cmake
Parsing /home/alkh-local/ncs/zephyr/tests/bluetooth/tester/Kconfig
Loaded configuration '/home/alkh-local/ncs/zephyr/boards/posix/native_posix/native_posix_defconfig'
Merged configuration '/home/alkh-local/ncs/zephyr/tests/bluetooth/tester/prj.conf'
Merged configuration '/home/alkh-local/ncs/zephyr/tests/bluetooth/tester/boards/native_posix.conf'
Merged configuration '/home/alkh-local/ncs/zephyr/tests/bluetooth/tester/overlay-mesh.conf'
Configuration saved to '/home/alkh-local/ncs/zephyr/tests/bluetooth/tester/build/zephyr/.config'
Kconfig header saved to '/home/alkh-local/ncs/zephyr/tests/bluetooth/tester/build/zephyr/include/generated/autoconf.h'
-- Found GnuLd: /usr/bin/ld.bfd (found version "2.38") 
-- The C compiler identification is GNU 11.4.0
-- The CXX compiler identification is GNU 11.4.0
-- The ASM compiler identification is GNU
-- Found assembler: /usr/bin/gcc
-- Configuring done (4.8s)
-- Generating done (0.0s)
-- Build files have been written to: /home/alkh-local/ncs/zephyr/tests/bluetooth/tester/build
-- west build: building application
[1/187] Preparing syscall dependency handling

[2/187] Generating include/generated/version.h
-- Zephyr version: 3.4.99 (/home/alkh-local/ncs/zephyr), build: zephyr-v3.4.0-4117-g146e7bbdb5ae
[187/187] Linking C executable zephyr/zephyr.elf
```
There is no built issue for `qemu_x86` either.

Reply:
[Try overlay-mesh-v1d1.conf](https://github.com/zephyrproject-rtos/zephyr/actions/runs/6310656031/job/17133047638)

Reply:
Ok, since it used only in dfu_cli there is no reason to make it common for mesh-1.0.1 and mesh-1.1